src/PMserial.h

```diff
@@ -128,6 +128,13 @@ class SerialPM
   inline bool has_number_concentration() { return (status == OK) && (pms != PMS3003); }
   inline bool has_temperature_humidity() { return (status == OK) && ((pms == PMS5003T) || (pms == PMS5003ST)); }
   inline bool has_formaldehyde() { return (status == OK) && ((pms == PMS5003S) || (pms == PMS5003ST)); }
+    
+  // adding offsets works well in normal range
+  // might introduce under- or overflow at the ends of the sensor range
+  void  set_rhum_offset(float offset) { rhum_offset = offset; };
+  void  set_temp_offset(float offset) { temp_offset = offset; };
+  float get_rhum_offset() { return rhum_offset; };
+  float get_temp_offset() { return temp_offset; };
```

Comment:
I think that this methods could be `inline`-ed
Also, I see the need for `set_rhum_offset`/`set_temp_offset` but I'm not so sure about `get_rhum_offset`/`get_temp_offset`